client/src/modules/multiple_payroll/modals/config.modal.html

```diff
@@ -1,79 +1,124 @@
-<form name="ConfigPaiementForm" bh-submit="ConfigPaiementModalCtrl.submit(ConfigPaiementForm)" novalidate>
+<form name="ConfigPaymentForm" bh-submit="ConfigPaymentModalCtrl.submit(ConfigPaymentForm)" novalidate>
   <div class="modal-header">
     <ol class="headercrumb">
       <li class="static" translate>TREE.HUMAN_RESOURCES</li>
-      <li class="title" translate>TREE.MULTI_PAYROLL</li>
-      <label class="badge badge-success">{{ ConfigPaiementModalCtrl.employee.display_name }}</label>
-      <label class="badge badge-primary"><span translate> {{ ConfigPaiementModalCtrl.label }} </span> : {{ ConfigPaiementModalCtrl.displayValue }}</label>
+      <li class="static" translate>TREE.MULTI_PAYROLL</li>
+      <li class="title" translate>{{::ConfigPaymentModalCtrl.period.label}}</li>
```

Comment:
This does not seem to work correctly.  Displays as '2', not USD, etc

Reply:
You are commenting on a line in the `config` modal, but I think you are referring to the $USD value in the filters for the `multiple_payroll` module.   If so, then that is a long-standing issue that won't be addressed in this PR. 

I'm not super concerned about it, since the user has other cues about the currency displayed.